css/main.css

```diff
@@ -34,6 +34,70 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type="number"] {
+	max-width: 7em;
+
+	/* Remove spinner controls from Firefox */
+	appearance: textfield;
+}
+
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type="number"]:hover,
+input[type="number"]:focus {
+	appearance: number-input;
+}
+
+select[data-provides="employee"] {
+	width: 6.5em;
+}
+
+/* Enables the first input's help text to pop
+over the lasts of this group.
+Also keeps related inputs together. */
+#input fieldset {
+	white-space: nowrap;
+	display: inline-block;
+}
+
+/* Reset fieldset style */
+fieldset {
+	border: 0;
+	padding: 0;
+	padding-top: .01em;
+	margin: 0;
+	min-width: 0;
+}
+
+input#salaire:hover ~ *,
+input#salaire:focus ~ * {
+	visibility: hidden;
+}
+input#salaire:hover + label,
+input#salaire:focus + label {
+	visibility: visible;
+}
+input#salaire:hover ~ .input-help,
+input#salaire:focus ~ .input-help {
+	visibility: visible;
+	opacity: 1;
+}
+
+input#salaire ~ .input-help {
+	visibility: hidden;
+	opacity: 0;
+	transition: opacity .1s linear;
+	display: inline-block;
+	font-size: 60%;
+	margin: 0 -22em 0 .5em;
```

Comment:
> This is very brittle

If you have another way of doing this, please don't hesitate ! I'll add a comment

This indication is hover text is fine until 500px wide. Which means it's not fine for most smartphones. A @media could be used later to move it below the input.